homeassistant/components/somfy/__init__.py

```diff
@@ -179,3 +179,4 @@ async def update_all_devices(hass):
         data[DEVICES] = await hass.async_add_executor_job(data[API].get_devices)
     except HTTPError as err:
         _LOGGER.warning("Cannot update devices: %s", err.response.status_code)
+        data[DEVICES] = []
```

Comment:
Allow to avoid the error `KeyError: 'devices'` during the setup.

Reply:
This also means that if fetching ever becomes an error, it will also override all devices.

Wouldn't it be better if we initialize this constant with an empty list and let errors not mess with it ?

Reply:
You are right, I think about it also and hesitate. I will make the modification.